src/workspace-mapping.ts

```diff
@@ -67,6 +68,7 @@ export function handleWorkspaceUpdated(event: WorkspaceUpdated): void {
   if(json.logoIpfsHash) entity.logoIpfsHash = json.logoIpfsHash!
   if(json.coverImageIpfsHash) entity.coverImageIpfsHash = json.coverImageIpfsHash
   if(json.socials) entity.socials = mapWorkspaceSocials(entityId, json.socials!)
+  if(json.tokens) entity.tokens = mapWorkspaceTokens(entityId, json.tokens!)
```

Comment:
Won't need to set "tokens" when "@derivedFrom" is set -- I believe this is mentioned in the graph's docs